gallery/streamlit_app.py

```diff
@@ -139,12 +141,8 @@ def get_empty():
 
 st.page(get_empty, "  ", " ")
 
-PATH_TO_EXTRAS = "streamlit_extras"
-extra_names = [
-    folder.name
-    for folder in Path(PATH_TO_EXTRAS).glob("*")
-    if folder.is_dir() and folder.name != "__pycache__"
-]
+
+extra_names = [extra.name for extra in pkgutil.iter_modules(streamlit_extras.__path__)]
```

Comment:
More elegant than me... https://github.com/arnaudmiribel/streamlit-extras/compare/main...refactor#diff-89043582f5d25e04fc6d4e5667995a4e8990c779185d80a7598da973c732385dR146

Reply:
Yeah, I really like this method, because it no longer depends on the particular structure of your code, it just depends on having the package